tests/modeling.py

```diff
@@ -0,0 +1,185 @@
+        if 'doubled' in key:
+            local_outputs = doubled_labelled_outputs
+        else:
+            local_outputs = labelled_outputs
+
+    params = hyperparameters[key]
+    dessia_models[key] = dessia_classes[key].fit_predict(std_inputs, local_outputs, std_inputs[50:100], **params)
+    dessia_models[key] = dessia_classes[key].fit(std_inputs, local_outputs, **params)
+    try:
+        assert(isinstance(dessia_models[key].score(std_inputs, local_outputs), float))
+    except ValueError as e:
+        assert(e.args[0] == 'multiclass-multioutput is not supported' and
+               isinstance(dessia_models[key], DecisionTreeClassifier))
+    dessia_models[key]._check_platform()
+
+
+# Tests errors and base objects
+base_scaler = Scaler()
+base_model = Model()
+base_linear = LinearModel()
+base_tree = Tree()
+base_rf = RandomForest()
+base_svm = SupportVectorMachine()
+base_mlp = MultiLayerPerceptron()
+
+try:
+    base_scaler._skl_class()
+    raise ValueError("_skl_class() should not work for Scaler object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for Scaler. Please use children.')
+
+try:
+    base_model._skl_class()
+    raise ValueError("_skl_class() should not work for Model object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for Model.')
+
+try:
+    base_linear._skl_class()
+    raise ValueError("_skl_class() should not work for LinearModel object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for LinearModel. Please use Ridge or LinearRegression.')
+
+try:
+    base_rf._skl_class()
+    raise ValueError("_skl_class() should not work for RandomForest object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for RandomForest. Please use RandomForestClassifier '\
+            'or RandomForestRegressor.')
+
+try:
+    base_svm._skl_class()
+    raise ValueError("_skl_class() should not work for SVM object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for SupportVectorMachine. Please use '\
+                              'SupportVectorClassifier or SupportVectorRegressor.')
+
+try:
+    base_mlp._skl_class()
+    raise ValueError("_skl_class() should not work for MultiLayerPerceptron object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _skl_class not implemented for MultiLayerPerceptron. Please use MLPRegressor '\
+            'or MLPClassifier.')
+
+try:
+    base_model._instantiate_skl()
+    raise ValueError("_instantiate_skl() should not work for Model object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _instantiate_skl not implemented for Model.')
+
+try:
+    base_model._instantiate_dessia(None)
+    raise ValueError("_instantiate_dessia() should not work for Model object.")
+except Exception as e:
+    assert (e.args[0] == 'Method _instantiate_dessia not implemented for Model.')
```

Comment:
What is tested? that some method raise NotImplemented Errors? Not really usefull and checking so precisely the error message seems dangerous